styleguide/source/_patterns/04-templates/two-column-left.json

```diff
@@ -2,7 +2,7 @@
   "two_column_left": {
     "utility": {
       "placeholder": {
-      "text": "Utility placeholder"
+        "text": "Utility placeholder"
```

Comment:
Was a tab added here?

Reply:
yes, shouldn't there be a tab here? since text is nested in placeholder?

Reply:
👍 wasn't thinking just saw the extra tab and was confused